### PR TITLE
Sections for client extensions and authenticator extensions

### LINKED ISSUE #1760: Seperate defined extensions by client and authenticator extensions.
Section 10, has a list of defined extensions. To aid clarity for RP's who wish to know which are client or authenticator, this could be re-arranged so that there is a client and authenticator subsection. An example of why this is an improvement is that currently it is "a detail" of the extension to if it is client or authenticator, and it's easy to overlook leading to implementor confusion (e.g. why is credProps not appearing in my authenticator extensions output.)

### PR DESCRIPTION
**Pull request by Firstyear**
Fixes #1760

***
<a href="https://pr-preview.s3.amazonaws.com/Firstyear/webauthn/pull/1762.html" title="Last updated on Jul 11, 2022, 10:38 PM UTC (6ea4140)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webauthn/1762/8ad9448...Firstyear:6ea4140.html" title="Last updated on Jul 11, 2022, 10:38 PM UTC (6ea4140)">Diff</a>

### CODE REVIEW

**Comment by Firstyear** [comment]
I've linked my w3c account now.

File: index.bs [hunk 1]
```diff
@@ -6026,8 +6026,9 @@ This section defines an additional set of extensions to be registered in the
 IANA "WebAuthn Extension Identifiers" registry [[!IANA-WebAuthn-Registries]] established by [[!RFC8809]].
```
Comment (emlun) [inline index.bs:6026]: Please amend the line above this (6025) to:

```
This section and its subsections define an additional set of extensions to be registered in the
```

File: index.bs [hunk 2]
```diff
@@ -6026,8 +6026,9 @@ This section defines an additional set of extensions to be registered in the
...
 These MAY be implemented by user agents targeting broad interoperability.
 
+## Client Extensions
```
Comment (emlun) [inline index.bs:6029]:
```suggestion
## Client Extensions

This section defines extensions that are only [=client extensions=].
```

**Review by emlun — CHANGES_REQUESTED** [review]
Thanks! This looks good to me apart from some minor polish suggestions.

**Comment by Firstyear** [comment]
Thanks @emlun, I've applied your feedback :)

File: index.bs [hunk 3]
```diff
@@ -6022,12 +6022,15 @@ There MUST NOT be any values returned for ignored extensions.
 
 # Defined Extensions # {#sctn-defined-extensions}
 
-This section defines an additional set of extensions to be registered in the
+This section and it's subsections defines an additional set of extensions to be registered in the
```
Comment (emlun) [inline index.bs:6025]: Typos:
```suggestion
This section and its subsections define an additional set of extensions to be registered in the
```

File: index.bs [hunk 4]
```diff
@@ -6022,12 +6022,15 @@ There MUST NOT be any values returned for ignored extensions.
...
 IANA "WebAuthn Extension Identifiers" registry [[!IANA-WebAuthn-Registries]] established by [[!RFC8809]].
 These MAY be implemented by user agents targeting broad interoperability.
 
+## Client Extensions
 
-## FIDO <dfn>AppID</dfn> Extension (appid) ## {#sctn-appid-extension}
+This section defines extensions that are only [=client extensions=].
+
+### FIDO <dfn>AppID</dfn> Extension (appid) ## {#sctn-appid-extension}
```
Comment (emlun) [inline index.bs:6033]: Ah right, you need to pad the header mark on both sides...
(this becomes very apparent in the [preview](https://pr-preview.s3.amazonaws.com/Firstyear/webauthn/pull/1762.html), which gets auto-added to the OP.)

```suggestion
### FIDO <dfn>AppID</dfn> Extension (appid) ### {#sctn-appid-extension}
```

File: index.bs [hunk 5]
```diff
@@ -6121,7 +6124,7 @@ Instead, in step three, the comparison on the host is relaxed to accept hosts on
 : Authenticator extension output
 :: None.
 
-## FIDO AppID Exclusion Extension (appidExclude) ## {#sctn-appid-exclude-extension}
+### FIDO AppID Exclusion Extension (appidExclude) ## {#sctn-appid-exclude-extension}
```
Comment (emlun) [inline index.bs:6127]:
```suggestion
### FIDO AppID Exclusion Extension (appidExclude) ### {#sctn-appid-exclude-extension}
```

File: index.bs [hunk 6]
```diff
@@ -6207,107 +6210,8 @@ During a transition from the FIDO U2F JavaScript API, a [=[RP]=] may have a popu
...
-                83              -- Item 2: CBOR array of length 3
-                    04           -- Subitem 1: CBOR integer for User Verification Method Passcode
-                    01           -- Subitem 2: CBOR short for Key Protection Type Software
-                    01           -- Subitem 3: CBOR short for Matcher Protection Type Software
-    </pre>
 
-
-
-## Credential Properties Extension (<dfn>credProps</dfn>) ## {#sctn-authenticator-credential-properties-extension}
+### Credential Properties Extension (<dfn>credProps</dfn>) ## {#sctn-authenticator-credential-properties-extension}
```
Comment (emlun) [inline index.bs:6214]:
```suggestion
### Credential Properties Extension (<dfn>credProps</dfn>) ### {#sctn-authenticator-credential-properties-extension}
```

File: index.bs [hunk 7]
```diff
@@ -6368,7 +6272,7 @@ At this time, one [=credential property=] is defined: the [=resident key credent
 :: None.
 
 
-## Large blob storage extension (<dfn>largeBlob</dfn>) ## {#sctn-large-blob-extension}
+### Large blob storage extension (<dfn>largeBlob</dfn>) ## {#sctn-large-blob-extension}
```
Comment (emlun) [inline index.bs:6275]:
```suggestion
### Large blob storage extension (<dfn>largeBlob</dfn>) ### {#sctn-large-blob-extension}
```

File: index.bs [hunk 8]
```diff
@@ -6482,6 +6386,110 @@ However, [=authenticators=] that do not utilize [[!FIDO-CTAP]] do not necessaril
 :: [=largeblob|This extension=] directs the user-agent to cause the large blob to be stored on, or retrieved from, the authenticator. It thus does not specify any direct authenticator interaction for [=[RPS]=].
 
 
+## Authenticator Extensions
+
+This section defines extensions that are both [=client extensions=] and [=authenticator extensions=].
+
+### <dfn>User Verification Method</dfn> Extension (uvm) ## {#sctn-uvm-extension}
```
Comment (emlun) [inline index.bs:6393]:
```suggestion
### <dfn>User Verification Method</dfn> Extension (uvm) ### {#sctn-uvm-extension}
```

**Review by emlun — CHANGES_REQUESTED** [review]

File: index.bs (hunk 1 above)
Comment (emlun) [reply index.bs:6026]: (The "defines" still needs to be updated to "define" here :slightly_smiling_face:)

**Comment by Firstyear** [comment]
Sorry about that, fixed up :)

**Pull request merged**